src/index.ts

```diff
@@ -95,10 +95,30 @@ module.exports = class ServerlessPlugin {
 
   private async afterPackageFunction() {
     const config = getConfig(this.serverless.service);
+    const forwarderArn: string | undefined = config.forwarderArn;
+    const forwarder: string | undefined = config.forwarder;
+
+    let functionArn;
     if (config.enabled === false) return;
-    if (config.forwarder) {
+
+    if (forwarderArn && forwarder) {
+      console.error(
+        "Error: Could not add CloudWatch forwarder subscriptions because both 'forwarderArn' and 'forwarder' parameters are set. 'forwarderArn' and 'forwarder' are equivalent, please only use 'forwarderArn' or 'forwarder'.",
+      );
+    } else if (forwarderArn !== undefined && forwarder === undefined) {
+      functionArn = forwarderArn;
+    } else if (forwarder !== undefined && forwarderArn === undefined) {
+      functionArn = forwarder;
+    } else {
+      console.error(
```

Comment:
I think it might be a better experience for customers who set both `forwarder` and `forwarderArn` for us to just favor `forwarderArn` and perhaps display a warning, rather than potentially deleting their existing log group subscriptions. Alternatively, maybe there is a way to make the deployment completely fail until the customer corrects the config? Either way, I don't think we should ever delete log group subscriptions because the customer adds a redundant ARN to their config.

Reply:
Hmm yeah this crossed my mind also but I couldn't find a way of terminating the deployment, I am going to investigate how I can terminate it with an error message telling the customer to correct the config.

Reply:
I edited the code so that if both forwarder and forwarderArn are set then a error will be thrown telling the customer to only use forwarderArn. I tested this with an existing forwarder subscription and the thrown error did not cause the subscription to be deleted, instead just the deployment failed. 
This is also the case of if no neither the forwarder nor the forwarderArn is provided. Only the current deployment will fail, no subscriptions will be lost anymore.